elements/chameleon-common/post-install.d/50-openrc

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+if [ ${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+# script for generating openrc file
+cat > /etc/auto_generate_openrc <<- EOM
+#!/bin/bash
+
+OPENSTACK_VENDOR_DATA='http://169.254.169.254/openstack/latest/vendor_data.json'
+OPENSTACK_VENDOR_DATA_2='http://169.254.169.254/openstack/latest/vendor_data2.json'
+
+JSON_VENDOR_DATA=\$(curl -s \$OPENSTACK_VENDOR_DATA_2)
+if [ "\$JSON_VENDOR_DATA" = '{}' ]; then
+  JSON_VENDOR_DATA=\$(curl -s \$OPENSTACK_VENDOR_DATA)
+fi
+
+function extract_json_key {
+  RESULT=\$(echo "\$2" | sed "s/.*\$1\": \"//g" | sed 's/".*//g')
+  echo "\$RESULT"
+}
+ 
+OPENRC=\$(extract_json_key "openrc" "\$JSON_VENDOR_DATA")
+SWIFTRC=\$(extract_json_key "swiftrc" "\$JSON_VENDOR_DATA")
+
+echo -e \$OPENRC >  /home/cc/openrc
+echo -e \$SWIFTRC >  /home/cc/swiftrc
```

Comment:
What is different about SWIFTRC? Can they not be the same?

Reply:
will update